src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private int radius;
+
+    public Circle(int radius, String figureColor) {
```

Comment:
```suggestion
    public Circle(int radius, String color) {
```
the same for other figures